utils/src/main/scala/dev/atedeg/mdm/utils/Range.scala

```diff
@@ -0,0 +1,27 @@
+package dev.atedeg.mdm.utils
+
+import scala.annotation.targetName
+import scala.math.Numeric.Implicits.*
+import scala.math.Ordering.Implicits.*
+
+final case class Range[T: Ordering] private[utils] (min: T, max: T)
+
+final case class RangePercentage private[utils] (percentage: Double)
+
+extension [T: Numeric](x: T)
+
+  def percent: RangePercentage = RangePercentage(x.toDouble / 100)
+
+  @targetName("plusMinus")
+  @SuppressWarnings(Array("org.wartremover.warts.Overloading"))
```

Comment:
Disable the wart directly in the `build.sbt`, we'll have to overload so many methods that it would become way too noisy to disable it one by one